server_addon/houdini/package.py

```diff
@@ -1,3 +1,12 @@
 name = "houdini"
 title = "Houdini"
-version = "0.2.15"
+version = "0.3.0"
+
+app_host_name = "houdini"
+
```

Comment:
My fault, it should be there, but `app_host_name` must be same for all addon versions. That changed in 1.1.1 server.
```suggestion
```